Description: Use babel7
 babel-preset-env is now @babel/preset-env
--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -43,7 +43,7 @@
       options: {
         sourceMaps: 'inline',
         presets: [
-          ['env', { loose: true }]
+          '@babel/env'
         ],
         plugins: [
           'add-module-exports'
